__tests__/data/package-dev-engines.json

```
@@ -0,0 +1,11 @@
+{
+  "engines": {
+    "node": "^20 || ^22"
```

this would make the test more reliable and easier to understand

```suggestion
    "node": "^19"
```